eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -21,6 +21,7 @@ parameters:
 jobs:
 - job: Validate
   condition: and(succeededOrFailed(), ne(variables['Skip.LiveTest'], 'true'))
+  timeoutInMinutes: 90
```

Comment:
Why was this necessary? And why are we picking 90 minutes?

Reply:
Default timeout for tests in devops is 60 min.  
Installing dependencies plus compiling the code (doing code analysis) with tests and finally running the test is taking more than one hour in the worst case for Windows.  The tests timeout before completing.  I am trying to set 90 min so we can finish

Reply:
Please add that as a comment justifying making the timeout larger than the default. 90 seems like a reasonable bump from 60. Thanks for clarifying. We should keep an eye on that though, we don't want really long runs (even if it is nightly) in the worst case.

Reply:
Yes, It would be usually less than that thanks to vcpkg cache, but on worst case, for windows, it really takes a long time to build and install dependencies and another long time building